--- mklive.sh.in	2019-01-14 21:36:21.861230656 -0500
+++ mklive.sh.in.new	2019-01-14 22:56:11.989702372 -0500
@@ -154,18 +154,30 @@
 generate_initramfs() {
     local _args
 
+    echo "generate_initramfs 1"
     copy_dracut_files "$ROOTFS"
+    echo "generate_initramfs 2"
     copy_autoinstaller_files "$ROOTFS"
+    echo "generate_initramfs 3"
     if [ "$BASE_SYSTEM_PKG" = "base-system-systemd" ]; then
         _args="--add systemd"
     else
         _args="--omit systemd"
     fi
+    echo "generate_initramfs 4"
+    echo "VOIDHOSTDIR: $VOIDHOSTDIR"
+    echo "ls VOIDHOSTDIR"
+    ls "$VOIDHOSTDIR"
+    echo "ls /sbin/bootchartd"
+    ls /sbin/bootchartd
     chroot "$ROOTFS" env -i /usr/bin/dracut -N --"${INITRAMFS_COMPRESSION}" \
         --add-drivers "ahci" --force-add "vmklive autoinstaller" ${_args} "/boot/initrd" $KERNELVERSION
+    echo "generate_initramfs 5"
     [ $? -ne 0 ] && die "Failed to generate the initramfs"
 
+    echo "generate_initramfs 6"
     mv "$ROOTFS"/boot/initrd "$BOOT_DIR"
+    echo "generate_initramfs 7"
     cp "$ROOTFS"/boot/vmlinuz-$KERNELVERSION "$BOOT_DIR"/vmlinuz
 }
 
